reference/release-notes/1.22.0.md

```diff
@@ -0,0 +1,83 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.22.0.
+
+Please see [component versions](https://anbox-cloud.io/docs/reference/component-versions) for a list of updated components.
+
+## New features & improvements
+
+The 1.22.0 release of Anbox Cloud brings the following features and improvements:
+
+### Core stack improvements
+* Support for 64-bit only ARM systems for all supported Android versions.<!--AC-1890, AC-2428-->
+* Support for installing relevant 32-bit NVIDIA userspace binaries on AMD64 systems.<!--AC-2450-->
```

Comment:
I am not sure if we want to mention this here as the installation of 32-bit NVIDIA drivers are meant to support the PC gaming(generic image)

Reply:
cc @morphis Do we want to mention this? FWIW, We have the messaging about custom image (not sure if you mean the same by generic image) for 1.22.0.

Reply:
We should mention that the 32bit driver binaries are installed as it is visible to the user. Lets not mention generic images here.

Reply:
so keeping as is.